frontend/server/src/DAO/SchoolOfTheMonth.php

```diff
@@ -192,6 +192,37 @@ public static function getSchoolsOfTheMonth(): array {
         return \OmegaUp\MySQLConnection::getInstance()->getAll($sql, []);
     }
 
+    /**
+     * @return \OmegaUp\DAO\VO\SchoolOfTheMonth[]
```

Comment:
```suggestion
     * @return list<\OmegaUp\DAO\VO\SchoolOfTheMonth>
```

esto quizás requira cambiar L221 para usar `$schools[] = ...;` en vez de `array_push()`. Igual en el otro archivo.